runtime/src/main/java/org/corfudb/runtime/object/ICorfuSMRProxyInternal.java

```diff
@@ -0,0 +1,20 @@
+package org.corfudb.runtime.object;
+
+import org.corfudb.protocols.logprotocol.SMREntry;
+
+import java.util.Map;
+
+/**
+ * Created by mwei on 11/15/16.
+ */
+public interface ICorfuSMRProxyInternal<T> extends ICorfuSMRProxy<T> {
+    VersionLockedObject<T> getUnderlyingObject();
```

Comment:
These methods need comments as to what they do. It is then easier for reviewers and other coders to figure out fixes/enhancements.